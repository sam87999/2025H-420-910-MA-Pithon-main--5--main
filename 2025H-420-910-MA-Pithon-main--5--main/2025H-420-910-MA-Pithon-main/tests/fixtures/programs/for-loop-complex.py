# Test de Maxime Bourret
numbers = [1, 2, 3, 4, 5]
total = 0
for num in numbers:
    total = total + num
print(total)

for x in (10, 20, 30):
    print(x * 2)

nested_list = [[1, 2], [3, 4]]
for sublist in nested_list:
    print(sublist) 