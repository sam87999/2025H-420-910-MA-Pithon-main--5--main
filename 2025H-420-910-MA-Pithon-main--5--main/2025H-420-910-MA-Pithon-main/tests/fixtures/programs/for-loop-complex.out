15
20
40
60
[1, 2]
[3, 4]
